docs/content/fin/custom-commands.md

```diff
@@ -199,6 +199,20 @@ Commands are ran in the same exact way as normal except include the folder they
 fin drupal/updb
 ```
 
+## Reusing fin functions
+
+To reuse some of the functionality within the `fin` command like `parse_params`, `is_linux`, or any of the functions it's possible to source the `fin` file.
```

Comment:
It is possible to source the `fin` file in order to reuse some of the functionality within the `fin` command like `parse_params`, `is_linux`, or any of the functions.